docs-archive/apache-airflow-providers-airbyte/1.0.0/index.html

```diff
@@ -331,7 +331,6 @@
 </ul>
 <p class="caption"><span class="caption-text">Resources</span></p>
 <ul>
-<li class="toctree-l1"><a class="reference external" href="https://github.com/apache/airflow/tree/master/airflow/providers/airbyte/example_dags">Example DAGs</a></li>
```

Comment:
Any idea what the Airbyte example DAGs get removed in v1.0.0, appear in v2.0.0 (pointing the expected tag), disappear again in v2.1.0, and then reappear in v2.1.2?

Reply:
I think it was because of some intermittent issue. I was checking if the URL is reachable and likely there was a temporary issue with GitHub at this moment. I did watch it while running so even if this happened, that would not be a "widesprad" issue. I would not worry too much about finding and fixing those issues though - at the worst case soneone does not find a link to a specific version's examples which is better than having a link that leads to "Missing".

I think it could have been done with only actually reacting to 404 (maybe this case was like 500. But it's a done deal now :)